topo_processor/util/aws_credentials.py

```diff
@@ -36,11 +37,13 @@ def init_roles() -> None:
     if aws_profile is None:
         return
 
+    get_log().debug("load_bucket_config", ssm=linz_ssm_bucket_config_name)
```

Comment:
Is there a reason for you writing log messages with underscores `_`?

Reply:
`s3_download` is a reference to the function `s3_download()`. Not a big deal, I was just curious :)

Reply:
Most of my other libs don't use spaces primarily so I can grep for them without quoting it, I would often use `TileCase`